modules/ssp_genieeBidAdapter.js

```diff
@@ -334,11 +335,10 @@ export const spec = {
    */
   isBidRequestValid: function (bidRequest) {
     if (!bidRequest.params.zoneId) return false;
-    if (bidRequest.params.hasOwnProperty('currency')) {
-      if (ALLOWED_CURRENCIES.indexOf(bidRequest.params.currency) === -1) {
-        utils.logError('Invalid currency type, we support only JPY and USD!');
-        return false;
-      }
+    const currencyType = config.getConfig('currency.adServerCurrency');
+    if (typeof currencyType === 'string' && ALLOWED_CURRENCIES.indexOf(currencyType) === -1) {
+      utils.logError('Invalid currency type, we support only JPY and USD!');
+      return false;
```

Comment:
>Do you need to add a currency param instead of using the adserverCurrency as you do in your dsp adapter ?

adserverCurrencyの使用で、リクエストしている貨幣種類を返してくれるとのことなので、こちらを使用するように修正

（ref: https://github.com/prebid/Prebid.js/blob/master/modules/dsp_genieeBidAdapter.js#L74 ）